osis-core/src/test/java/com/scality/osis/utils/ScalityTestUtils.java

```diff
@@ -20,6 +20,9 @@
                     "5b7e3259-aace-414c-bfd8-94daa0efefaf","6b7e3259-aace-414c-bfd8-94daa0efefaf",
                     "4b7e3259-aace-414c-bfd8-94daa0efefaf","3b7e3259-aace-414c-bfd8-94daa0efefaf")
     );
+    public static final String TEST_ROLE_ARN = "arn:aws:iam::12313265465789:role/osis";
```

Comment:
Not a good account ID.
Account ID is 12 digits.

Reference: https://github.com/scality/Vault/blob/development/8.2/lib/IdGenerator.js#L17

Reply:
done in 771e556f7555e54231774c097bd91a7c8688fd1a